### PR TITLE
Add European Structural and Investment fund doctype

### PR DESCRIPTION
**Pull request by jackscotti**
ticket: https://www.pivotaltracker.com/story/show/87188568

### CODE REVIEW

**Comment by evilstreak** [comment]
We want the document type to be `european_structural_investment_fund`, so that should be the name of the file.

**Comment by jackscotti** [comment]
I changed it with Tommy in order to follow the naming conventions:
https://github.com/alphagov/specialist-publisher/pull/414

**Comment by jackscotti** [comment]
@evilstreak you are right. I just renamed the file. Better to have verbose name files.

File: config/schema/default/doctypes/european_structural_investment_fund.json [hunk 1]
```diff
@@ -0,0 +1,169 @@
+{
+  "properties": {
+    "fund_state": {
+      "type": "string",
+      "index": "not_analyzed",
+      "include_in_all": false,
+      "details": {
+        "name": "Fund state",
+        "ui_widget_type": "single-select",
```
Comment (tommyp) [inline config/schema/default/doctypes/european_structural_investment_fund.json]: You don't need `ui_widget_type` in these schemas.

**Pull request merged**